embeddings/data/data_loader.py

```diff
@@ -34,29 +33,30 @@ def load(self, dataset: Dataset[str]) -> datasets.DatasetDict:
 
 class HuggingFaceLocalDataLoader(DataLoader[str, datasets.DatasetDict]):
     def load(self, dataset: Dataset[str]) -> datasets.DatasetDict:
-        if isinstance(dataset, HuggingFaceDataset):
-            result = datasets.load_from_disk(str(dataset.dataset))
+        if isinstance(dataset, LoadableDataset):
+            result = datasets.load_from_disk(dataset.dataset)
             assert isinstance(result, datasets.DatasetDict)
             return result
         else:
             raise ValueError("This DataLoader should be used with HuggingFaceDataset only.")
```

Comment:
is this error to be adjusted and handled?

Reply:
Yes. I removed it as it's no longer needed. In current version of dataloaders we do not differentiate that